skops/_min_dependencies.py

```diff
@@ -11,6 +11,7 @@
 #     "tomli": ("1.1.0", "install", "python_full_version < '3.11.0a7'"),
 dependent_packages = {
     "scikit-learn": ("0.24", "install", None),
+    "scikit-learn-intelex": ("2023.0.1", "docs", None),
```

Comment:
do we really need a very recent release here? These are minimum version dependencies.

Reply:
tbh: I don't know, so I added what I tested. We can go with a 1y old release. As far as I can tell this should work too. But I wouldn't want to put an even older release there.